documentation/modules/con-kafka-jmx-remote-options.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-kafka-jmx-options.adoc
+
+[id='con-kafka-jmx-options-{context}']
+
+= JMX remote options
+
+.Prerequisites
+
+* A Kubernetes cluster
+* A running Cluster Operator
+
+Use the `jmxRemote` property in the Kafka cluster spec to expose the JMX port on Kafka brokers via the Kafka broker headless service.
+You can enable security on the broker by turning on security in the `jmxRemote` spec.
+
+You can gather JMX metrics from the Kafka brokers by using the JMX options. For examples of JMX metrics that can be obtained,
```

Comment:
Could you please avoid having two sentences per line? It makes it easier to diff and comment on it in future changes.